src/__test__/reducers/assetLogs.reducer.test.js

```diff
@@ -0,0 +1,42 @@
+// third-party library
```

Comment:
Not sure what this comment is for but the code is self-explanatory so comment not needed